ccd-definition/CaseEventToFields/DefendantResponse-nonprod.json

```diff
@@ -49,7 +52,7 @@
     "DisplayContext": "MANDATORY",
     "PageID": "RespondentResponseType",
     "ShowSummaryChangeOption": "Y",
-    "FieldShowCondition": "respondentResponseIsSame != \"Yes\""
+    "FieldShowCondition": "respondent2SameLegalRepresentative = \"Yes\" AND respondentResponseIsSame != \"Yes\""
```

Comment:
this actually breaks 1v2 (diff solicitors) as second solicitor will not be able to see field. respondent2SameLegalRepresentative will be No. Why was this added?

Reply:
Needs to be removed from respondent2ClaimResponseType, respondent2PartyNameLabel and potentially applicant1PartyNameLabel2. The other show hides are fine, but should have worked without them